byway/pom.xml

```diff
@@ -42,6 +42,12 @@
       <artifactId>appengine-api-1.0-sdk</artifactId>
       <version>1.9.59</version>
     </dependency>
+
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.6</version>
+    </dependency>
```

Comment:
Looks like gson is already included in the version from master. You should probably just reset this file back to the way it is in master (with `git checkout master byway/pom.xml` while on this branch).

Reply:
Okay, I ran that command. The online merge tool just caused more conflicts.